_includes/templates/nav-about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
```

Comment:
Ah, sorry, I misunderstood the situation. I just saw "autoplay" and my canine instincts kicked in ;)